.github/workflows/api-test.yaml

```diff
@@ -0,0 +1,48 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Run API Service Tests
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ '**' ]
+jobs:
+  test:
+    runs-on: ubuntu-latest
```

Comment:
I searched a bit but couldn't find a definitive answer. Does GitHub actions have a caching policy on docker images?

Reply:
Really not an expert. I'm not sure if it's a big deal if we're not using something exotic. My latest run took 11s to setup the job, the actions, and the jdk, which seems fine. https://github.com/DataBiosphere/tanagra/runs/3116730025
Most of the caching literature seems to be about what's pulled into the environment, not so much the running docker image itself, https://docs.github.com/en/actions/guides/caching-dependencies-to-speed-up-workflows